mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/MessageMapper.java

```diff
@@ -207,7 +207,7 @@ default Mono<List<UpdatedFlags>> resetRecentReactive(Mailbox mailbox) {
     MessageMetaData copy(Mailbox mailbox, MailboxMessage original) throws MailboxException;
 
     default List<MessageMetaData> copy(Mailbox mailbox, List<MailboxMessage> original) throws MailboxException {
-        return original.stream()
+        return original.parallelStream()
```

Comment:
parrallelStream is a dangerous abstraction, we should not use it. Doing one range at the time is likely enough BTW.

If you really want to do stuff parrallely migrate this wall chunk of code to reactor.